src/srchd.ts

```diff
@@ -136,6 +137,9 @@ agentCmd
     "-c, --count <number>",
     "Number of agents to create (name used as prefix)",
   )
+  .option("--tools <tools>", "Tools to use (comma separated)", (value) =>
```

Comment:
Do not use comma separated I want --tool xx --tool xx

Default tools goal_solution, publications